src/app/lib/config/services/korean.js

```diff
@@ -32,6 +32,9 @@ const service = {
   manifestPath: '/manifest.json',
   swPath: '/sw.js',
   translations: {
+    home: '뉴스',
+    currentPage: 'Current page',
```

Comment:
For some services, we do not yet have all the translations. I have added those that we have available to this PR. Once we have the remaining translations, I'll open a PR to add them.